README.md

```diff
@@ -79,7 +79,7 @@ To run a specific problem in your console run  `node <problem_file_path>` (e.g.
 | [Permutations With Duplicates ](/LeetcodeProblems/Algorithms/Permutations_With_Duplicates.js)                   |        | |
 | [Deletion Distance](/LeetcodeProblems/Algorithms/Deletion_Distance.js)                                          |        | |
 | [Award Budget Cuts](/LeetcodeProblems/Algorithms/Award_Budget_Cuts.js)                                          |        | |
-
+| [Maximum Sum of an Hourglass](/LeetcodeProblems/Algorithms/Maximise_Hour_Glass_Sum.js)                          | Medium | https://leetcode.com/problems/maximum-sum-of-an-hourglass/ |
```

Comment:
Nit: The problem is sorted by complexity, could you move this problem to the medium section? Thanks!

Reply:
Updated the ordering @ignacio-chiazzo